src/components/SuiteHeader/_suite-header.scss

```
@@ -59,6 +59,7 @@
   right: 0;
   max-width: rem(425px);
   width: auto;
+  z-index: 1;
```

There's a z-index function that we can use to keep values consistent, it takes variables as params - [here's the source/list](
https://github.com/carbon-design-system/carbon/blob/73d94961ba828afc72fbb82a853d304caffbd023/packages/components/src/globals/scss/_layout.scss#L14-L23)


```suggestion
  z-index: z('overlay');
```

---

Thanks for pointing this out @tay1orjones 
I didn't know about this `z-index` function 👍